app/views/hiring_staff/vacancies/_sidebar.html.erb

```diff
@@ -0,0 +1,42 @@
+<% @steps_to_display = steps_to_display -%>
```

Comment:
I think we should standardise on HAML for the views, it is very easy to convert HTML to HAML with

https://html2haml.herokuapp.com

Reply:
I'm afraid I'm going to have to insist this remains ERB. The work is already done and is lifted *exactly* from Chris' prototype template. Converting it adds complexity, moves the code away from a web standard towards a Rails-only DSL, introduces the possibility of misinterpretation and raises the overall maintenance cost over time.